lib/ecto/query/builder.ex

```diff
@@ -583,13 +591,24 @@ defmodule Ecto.Query.Builder do
   @doc """
   Called by escaper at runtime to verify binaries.
   """
+  def unsafe_fragment!(string) do
+    unless is_binary(string) do
+      raise ArgumentError, bad_unsafe_fragment_message(inspect(string))
+    end
+
+    [{:raw, string}]
+  end
+
+  @doc """
+  Called by escaper at runtime to verify binaries.
+  """
   def unsafe_fragment!(string, fragments) do
     if is_binary(string) do
       pieces = split_fragment(string, "")
 
       if length(pieces) != length(fragments) + 1 do
         raise ArgumentError,
-              "unsafe_fragment(...) expects extra arguments in the same amount of question marks in string"
+              "unsafe_fragment/2 expects to have as many fragment elements as there are question marks in string"
```

Comment:
I have seen you renamed `unsafe_fragment(...)` to `unsafe_fragment` almost everywhere except here where you used `unsafe_fragment/2`. Maybe we should use `unsafe_fragment/2` consistently?